migrate/multi-node-to-timescale-instance.md

```diff
@@ -0,0 +1,189 @@
+
+The following instructions show you how to move your data from a self-hosted
+TimescaleDB Multi-node deployment to a Timescale instance using `pg_dump` and `psql`.
+To not lose any data, applications which connect to the database should be taken
+offline. The duration of the migration is proportional to the amount of data
+stored in your database. Should you wish to migrate more than 400 GB of data with
+this method, open a support request to ensure that enough disk is pre-provisioned
+on your Timescale instance.
+
+<OpenSupportRequest />
+
+<SourceTargetNote />
+
+## Prerequisites
+
+For minimal downtime, the `pg_dump` and `psql` commands should be run from a machine
+with a low-latency, high-throughput link to the database that they are connected to.
+
+Before you begin, ensure that you have:
+
+- Installed the PostgreSQL client libraries on the machine that you will
+  perform the migration from, you will require `pg_dump` and `psql`.
+- [Created a database service in Timescale].
+- Checked that all PostgreSQL extensions you use are available on Timescale.
+  For more information, see the [list of compatible extensions].
+- Checked that the version of PostgreSQL in your target database is greater
+  than or equal to that of the source database.
+- Checked that you're running the exact same version of Timescale on both your
+  target and source databases (the major/minor/patch version must all be the
+  same). For more information, see the [upgrade instructions] for self-hosted
+  TimescaleDB.
+
+[Created a database service in Timescale]: /use-timescale/:currentVersion:/services/create-a-service/
+[list of compatible extensions]: /use-timescale/:currentVersion:/extensions/
+[upgrade instructions]: /self-hosted/:currentVersion:/upgrades/about-upgrades/
+
+## Migrate roles from source to target
+
+<DumpDatabaseRoles />
+
+<SetupSourceTarget />
+
+## Migrate schema pre-data
+
+Migrate your pre-data from your source database to TimescaleDB instance. This
+includes table and schema definitions, as well as information on sequences,
+owners, and settings. This doesn't include Timescale-specific schemas.
+
+<Procedure>
+
+### Migrating schema pre-data
+
+1.  Dump the schema pre-data from your source database into a `dump_pre_data.bak` file, using
+    your source database connection details. Exclude Timescale-specific schemas.
+    If you are prompted for a password, use your source database credentials:
+
+    ```bash
+    pg_dump -U <SOURCE_DB_USERNAME> -W \
+    -h <SOURCE_DB_HOST> -p <SOURCE_DB_PORT> -Fc -v \
+    --section=pre-data --exclude-schema="_timescaledb*" \
+    -f dump_pre_data.bak <DATABASE_NAME>
+    ```
+
+1.  Restore the dumped data from the `dump_pre_data.bak` file into your Timescale
+    database, using your Timescale connection details. To avoid
+    permissions errors, include the `--no-owner` flag:
+
+    ```bash
+    pg_restore -U tsdbadmin -W \
+    -h <HOST> -p <PORT> --no-owner -Fc \
```

Comment:
Not sure if you need `-Fc ` here.

Reply:
I think you need it. For pg_restore to know the format of the dump